packages/expo-local-authentication/android/src/main/java/expo/modules/localauthentication/LocalAuthenticationModule.java

```diff
@@ -32,6 +33,9 @@
   private boolean mIsAuthenticating = false;
   private ModuleRegistry mModuleRegistry;
   private UIManager mUIManager;
+  private String mPromptMessage = "";
```

Comment:
Could you confirm that we use the same default message on iOS?

Reply:
Yep. It's the same.